src/Uno.UWP/Storage/StorageFolder.native.wasm.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using Uno.Foundation;
+using Windows.Foundation;
+
+namespace Windows.Storage
+{
+	public partial class StorageFolder
+	{
+		internal static StorageFolder CreateNativeHandler(string path, Guid guid)
```

Comment:
To match the UWP static methods (`GetFolderFrom***`), I would name it something like `GetFolderFromNativePathAsync`